src/stactools/viirs/fragment.py

```diff
@@ -69,6 +69,15 @@ def collection_dict(self) -> Dict[str, Any]:
         collection["links"] = [Link.from_dict(link) for link in collection["links"]]
         return collection
 
+    def collection_eo_bands_summary(self) -> List[Dict[str, Any]]:
+        """Returns a list of all Asset eo:bands for inclusion in Collection
+        summaries."""
+        summary = []
+        for asset in self.assets.values():
+            if "eo:bands" in asset:
+                summary.append(asset["eo:bands"][0])
```

Comment:
```suggestion
                summary.extend(asset["eo:bands"])
```

I think the (very low) performance hit of `extend` is worth future-proofing just in case we handle multi-band assets in the future (unlikely). #edgecases

Reply:
Yep. Done.